src/Options/FixtureOptions.php

```diff
@@ -300,14 +300,13 @@ public function getProjectTemplate(): string {
       return $this->options['project-template'];
     }
 
+    // @todo Point to latest version of DRP once 2.x for D10 stable fixtures
+    //   and "dev-master" for D10 dev fixtures.
     if ($this->coreVersionParsedMatches('^10')) {
       $this->options['project-template'] = 'acquia/drupal-recommended-project:dev-drupal10';
     }
-    elseif ($this->coreVersionParsedMatches('^9') && $this->isDev()) {
-      $this->options['project-template'] = 'acquia/drupal-recommended-project:dev-master';
```

Comment:
@secretsayan If we want to support D9 dev, we can use [dev-1.x](https://github.com/acquia/drupal-recommended-project/tree/1.x) branch.

Reply:
Currently we do not have any job that would point to D9 dev.


<img width="1158" alt="Screenshot 2023-01-19 at 7 18 52 PM" src="https://user-images.githubusercontent.com/9529657/213459263-c227fcca-700d-4c8d-8079-5ed53b1178b5.png">

Reply:
Ok thanks, make sense. 👍